source/opt/eliminate_dead_io_components_pass.h

```diff
@@ -26,10 +26,10 @@ namespace spvtools {
 namespace opt {
 
 // See optimizer.hpp for documentation.
-class EliminateDeadInputComponentsPass : public Pass {
+class EliminateDeadIOComponentsPass : public Pass {
  public:
-  explicit EliminateDeadInputComponentsPass(bool output_instead = false,
-                                            bool vertex_shader_only = true)
+  explicit EliminateDeadIOComponentsPass(bool output_instead = false,
```

Comment:
Still up to replace the output_instead by the storage class?
(https://github.com/KhronosGroup/SPIRV-Tools/pull/4984#discussion_r1018960530)

Reply:
Yes. Sorry! Realized as I was heading out yesterday that I forgot this. Came in quick on my holiday today to fix :)

Reply:
Oh yes US, happy thanksgiving!😊